docs/content/Examples-Tutorials-Recipes/Getting-Unique-Values-for-a-Field.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting Unique Values for a Field
+permalink: /recipes/getting-unique-values-for-a-field
+category: Examples & Tutorials
+subCategory: Queries
+menuOrder: 5
+---
+
+## Use case
+
+We have an internal analytical dashboard with information about the users of our
+online store. To filter users by city, we need to display all unique values in
+the dropdown. In the recipe below, we'll learn how to get unique values from all
+[dimensions](https://cube.dev/docs/schema/reference/dimensions).
+
+## Data schema
+
+To filter users by city, we need to define the appropriate dimension:
+
+```javascript
+cube(`Users`, {
+  sql: `SELECT * FROM public.users`,
+
+  dimensions: {
+    id: {
+      sql: `id`,
+      type: `number`,
+      primaryKey: true,
+    },
```

Comment:
I don't see this dimension in the data schema file. @rchkv, you're gonna sync them, right?

Reply:
😼 you're right! I removed `id` in the last commit and forgot update this page.